test/sql/test_agg_function/T/test_percentile

```diff
@@ -0,0 +1,30 @@
+-- name: testPercentileParameter
+CREATE TABLE IF NOT EXISTS `lineorder` (
+    `lo_orderkey` int(11) NOT NULL COMMENT "",
+    `lo_shipmode` varchar(11) NOT NULL COMMENT ""
+) ENGINE=OLAP
+DUPLICATE KEY(`lo_orderkey`)
+COMMENT "OLAP"
+DISTRIBUTED BY HASH(`lo_orderkey`) BUCKETS 48
+PROPERTIES (
+    "replication_num" = "1"
+);
+insert into lineorder values(1,'test');
+SELECT percentile_cont(lo_orderkey, -1) FROM lineorder;
+SELECT percentile_disc(lo_orderkey, -1) FROM lineorder;
+SELECT percentile_approx(lo_orderkey, -1) FROM lineorder;
+
+-- name: testPercentileEmptyTable
+CREATE TABLE IF NOT EXISTS `lineorder` (
+    `lo_orderkey` int(11) NOT NULL COMMENT "",
+    `lo_shipmode` varchar(11) NOT NULL COMMENT ""
+) ENGINE=OLAP
+DUPLICATE KEY(`lo_orderkey`)
+COMMENT "OLAP"
+DISTRIBUTED BY HASH(`lo_orderkey`) BUCKETS 48
+PROPERTIES (
+    "replication_num" = "1"
+);
+SELECT percentile_cont(lo_orderkey, -1) FROM lineorder;
+SELECT percentile_disc(lo_orderkey, -1) FROM lineorder;
+SELECT percentile_approx(lo_orderkey, -1) FROM lineorder;
```

Comment:
it seems that "SELECT percentile_cont(lo_orderkey,  lo_discount) FROM lineorder"  would cause be's crash. please test. 

Allow a const expr not evalauted in FE to be delivered to BE,  would leads to non-const expr to delivered to BE to, so why do we need to support a variable parameter in percentile_approx?   if we need to do so, suggest to only support const expr that can be reduced in FE, then check it in ScalarOperatorToExpr::visitCall